src/analysis/souffle/examples/multimic/multimic.arcs

```diff
@@ -0,0 +1,59 @@
+// Note: This file marks speculative syntax with a SPECULATIVE comment before
+// the speculative syntax.
+
+// Represents an actual microphone within a smart mic listening to audio.
+particle MicIn
+  audio: writes AudioData { }
+// SPECULATIVE: claim a QoS integrity tag.
+//claim audio is userSpeech and is integrity#speechToTextQoS
```

Comment:
What is the motivation for the integrity tags?

Reply:
I'm trying to express in the Arcs Manifest language a user's desire that the data path turning raw audio into a text transcript of speech stored upon the user's smart mic is not interrupted. A guest policy could introduce restrictions at multiple points along this path that could interrupt this path, so we verify that this path exists by proving that a tag may flow from the data source to the storage device.

Now, why an integrity tag vs a secrecy tag? Let's expand the example and say that the microphone's processing is, in addition to ASR data, some microphone health monitoring data, like which frequencies it has heard lately to ensure that the microphone is working properly. Imagine that this is stored in the same store as the actual ASR data. In this case, a user policy might interrupt the critical text to speech pipeline with their policy but leave the health metrics, through which the secrecy QoS tag could pass and cause the check to falsely pass, causing a false negative on the QoS check.

I know that we can work around that with separating the stores or adding `derives from` claims. That is probably what we will do until we have integrity tags. But to use the secrecy tags in this way, we'd have to actively work to pen the secrecy QoS tag onto the path that we want to prevent false negatives. So, long term, the propagation behavior of an integrity tag seems like a better fit for ensuring the type of property that we are trying to guarantee.

Reply:
Added an annotation which indicates that the original user may attempt to block dataflow to accomodate UserC's policy. In this case, however, the QoS tag would also be interrupted, ending in a new check, forcing the discarding of UserC's policy.